### PR TITLE
add package info and read me to the sql isolated worker

### PR DESCRIPTION
**Pull request by MaddyDev**

### CODE REVIEW

File: Directory.Build.props [hunk 1]
```diff
@@ -11,5 +11,22 @@
...
+		<IncludeSymbols>true</IncludeSymbols>
+		<SymbolPackageFormat>snupkg</SymbolPackageFormat>
+		<DebugType>portable</DebugType>
+		<PackageId>Microsoft.Azure.Functions.Worker.Extension.Sql</PackageId>
+		<PackageRequireLicenseAcceptance>True</PackageRequireLicenseAcceptance>
+		<PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+		<PublishRepositoryUrl>true</PublishRepositoryUrl>
+		<EmbedUntrackedSources>true</EmbedUntrackedSources>
```
Comment (Charles-Gagnon) [inline Directory.Build.props:27]: fixed spacing and removed package ID (that's unique per package)
```suggestion
    <Company>Microsoft</Company>
    <Authors>Microsoft</Authors>
    <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
    <DebugSymbols>true</DebugSymbols>
    <IncludeSymbols>true</IncludeSymbols>
    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
    <DebugType>portable</DebugType>
    <PackageRequireLicenseAcceptance>True</PackageRequireLicenseAcceptance>
    <PackageLicenseExpression>MIT</PackageLicenseExpression>
    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
    <PublishRepositoryUrl>true</PublishRepositoryUrl>
    <EmbedUntrackedSources>true</EmbedUntrackedSources>
```

File: Worker.Extension.Sql/src/Microsoft.Azure.Functions.Worker.Extension.Sql.csproj [hunk 2]
```diff
@@ -4,18 +4,17 @@
...
-		<!--Temporarily opting out of documentation. Pending documentation-->
-		<GenerateDocumentationFile>false</GenerateDocumentationFile>
 	</PropertyGroup>
 
 	<ItemGroup>
-		<PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Abstractions"/>
+		<PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Abstractions" />
 		<PackageReference Include="Microsoft.Data.SqlClient" />
 		<PackageReference Include="System.Drawing.Common" />
+		<None Include="pkgicon.png" Pack="true" PackagePath="" />
```
Comment (Charles-Gagnon) [inline Worker.Extension.Sql/src/Microsoft.Azure.Functions.Worker.Extension.Sql.csproj:16]: These can be added to the Build.Props too

File: Worker.Extension.Sql/src/Microsoft.Azure.Functions.Worker.Extension.Sql.csproj [hunk 3]
```diff
@@ -4,18 +4,17 @@
 		<AssemblyName>Microsoft.Azure.Functions.Worker.Extension.Sql</AssemblyName>
 		<RootNamespace>Microsoft.Azure.Functions.Worker.Extension.Sql</RootNamespace>
 		<Description>Sql extension for .NET isolated Azure Functions</Description>
-
+		<Product>SQL Binding Worker</Product>
```
[the remark this answers is not in the document]
Comment (MaddyDev) [reply Worker.Extension.Sql/src/Microsoft.Azure.Functions.Worker.Extension.Sql.csproj:7]: good catch, added it and removed the common ones from extension project file, thanks!

File: Directory.Build.props [hunk 4]
```diff
@@ -11,5 +11,20 @@
     <SignAssembly>True</SignAssembly>
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)/SQL2003.snk</AssemblyOriginatorKeyFile>
     <IsPackable>false</IsPackable>
+    <Company>Microsoft</Company>
+    <Authors>Microsoft</Authors>
+    <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
+    <IsPackable>true</IsPackable>
```
Comment (Charles-Gagnon) [inline Directory.Build.props:17]: This should be false in this file - we want projects to specifically opt-in to being packable in their own csproj

(we may want to look into only adding the nuget related properties when IsPackable is true, but I don't think there should be any harm in including them even if the project isn't being packaged so leaving them like this is probably fine)

File: src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj [hunk 5]
```diff
@@ -8,22 +8,10 @@
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <!-- Default Version for dev -->
     <Version>99.99.99</Version>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```
Comment (Charles-Gagnon) [inline src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj:11]: There's still some of the properties you added in the root (Company, Authors, etc.) in here. Make sure you go through all the ones you moved and remove them from here too.

File: Directory.Build.props [hunk 6]
```diff
@@ -11,5 +11,20 @@
     <SignAssembly>True</SignAssembly>
     <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)/SQL2003.snk</AssemblyOriginatorKeyFile>
     <IsPackable>false</IsPackable>
+    <Company>Microsoft</Company>
+    <Authors>Microsoft</Authors>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <IsPackable>false</IsPackable>
```
Comment (Charles-Gagnon) [inline Directory.Build.props:17]: You have this property defined twice now. Please remove the duplicate

**Pull request merged**